.github/workflows/ci.yml

```
@@ -55,6 +55,51 @@ jobs:
       - name: Run tests
         run: cargo test --features "cachekitio,redis,encryption,l1,macros"
 
+  redis-lock:
+    name: redis lock integration
+    # `services:` containers require Docker. cachekit-lean is a no-dind
+    # scaleset (lab ADR-0001), so a redis service can only run on the
+    # with-dind `cachekit` scaleset. Without a live redis the lock
+    # acquire/release path has zero automated coverage — the
+    # redis_lock_live_semantics test skips when CACHEKIT_TEST_REDIS_URL is
+    # unset (LAB-426: the AC requires locking exercised against a container).
+    runs-on: cachekit
```

actionlint (1.7.12)

[error] label "cachekit" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)